### PR TITLE
modules/nixos: github-org-backup 

### PR DESCRIPTION
**Pull request by zowoq**
Now that we have access to external storage we could use it to backup the repos.

https://github.com/gabrie30/ghorg

~~Still need to add secrets, I'll do that if we want to move forward with this.~~

### CODE REVIEW

File: services/github-backup/default.nix [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  systemd.services.github-backup = {
+    script = ''
+      GHORG_GITHUB_TOKEN="$(<$CREDENTIALS_DIRECTORY/github-backup-token)" \
+        ${pkgs.lib.getExe pkgs.ghorg} clone nix-community \
+        --backup \
+        --clone-wiki \
+        --concurrency 2 \
+        --exclude-match-regex '^nixpkgs$' \
+        --exclude-match-regex '^nix$' \
+        --path /var/lib/github-backup \
```
Comment (Mic92) [inline services/github-backup/default.nix:14]: Since this is public information, is there a way to make this backup public accessible?

Comment (zowoq) [reply services/github-backup/default.nix:14]: Do you mean serving this folder with nginx or something else?

Comment (Mic92) [reply services/github-backup/default.nix:14]: Something like that. I just don't know how easy it is to download the backup over nginx.

Comment (zowoq) [reply services/github-backup/default.nix:14]: Can you explain why you want the backups to be publicly accessible? My understanding was running a mirror wasn't the objective, it was just a way of being able to have backups?

**Comment by zowoq** [comment]
I'll finish

File: services/github-backup/default.nix (hunk 1 above)
Comment (adisbladis) [reply services/github-backup/default.nix]: We could expose the storage box from `web01` if we want it publicly accessible.

Comment (Mic92) [reply services/github-backup/default.nix]: with its own subdomain.

Comment (zowoq) [reply services/github-backup/default.nix]: > Can you explain why you want the backups to be publicly accessible? My understanding was running a mirror wasn't the objective, it was just a way of being able to have backups?

I'd still like an answer to this?

**Pull request merged**